cmd/azure-keyvault-controller/controller/secret_handler_test.go

```diff
@@ -46,7 +46,7 @@ func (f *fakeVaultService) GetSecret(secret *akv.AzureKeyVault) (string, error)
 func (f *fakeVaultService) GetKey(secret *akv.AzureKeyVault) (string, error) {
 	return "", nil
 }
-func (f *fakeVaultService) GetCertificate(secret *akv.AzureKeyVault, exportPrivateKey bool) (*vault.Certificate, error) {
+func (f *fakeVaultService) GetCertificate(secret *akv.AzureKeyVault, exportPrivateKey bool, ensureServerFirst bool) (*vault.Certificate, error) {
```

Comment:
Introduce a option struct, as mentioned earlier?

Reply:
Yes, I agree, will make the changes.